R/pkg/R/sparkR.R

```diff
@@ -410,6 +410,30 @@ sparkR.session <- function(
   sparkSession
 }
 
+#' Get the URL of the SparkUI instance for the current active SparkSession
+#'
+#' Get the URL of the SparkUI instance for the current active SparkSession.
```

Comment:
actually, no, the first is the title, then 2nd (after the empty line) is the description. we have that in some of our doc where there isn't really any more to say ;)